src/status_im2/contexts/activity_center/view.cljs

```diff
@@ -123,47 +124,59 @@
     [rn/view {:style style/filter-toggle-container}
      [filter-selector-read-toggle]]]])
 
-(defn render-notification
-  [{:keys [type] :as notification} index]
-  [rn/view {:style (style/notification-container index)}
-   (cond
-     (= type types/contact-verification)
-     [contact-verification/view notification {}]
+(defn notification-component
+  [_ _ _ _]
```

Comment:
Why are all the args "_" ?

Reply:
It seems I'm trying to draw a figure or something using only underscores... :monkey:

They're pointless, thanks for pointing that out!

Side note: In Clojure JVM, as is the case of pretty much all mainstream languages (except JS), the function arity must be respected, otherwise the code doesn't even compile. I guess sometimes I still forget about this. In JS it's more permissible as callers can pass more arguments to the functions or just omit them :man_shrugging: In ClojureScript we inherit this bad characteristic from the host language.

Reply:
Gone they are!